webapp/channels/src/components/admin_console/permission_schemes_settings/edit_post_time_limit_modal/edit_post_time_limit_modal.tsx

```diff
@@ -170,10 +170,17 @@ export default function EditPostTimeLimitModal(props: Props) {
                     onClick={save}
                     disabled={saving}
                 >
-                    <FormattedMessage
-                        id={saving ? t('save_button.saving') : t('edit_post.time_limit_modal.save_button')}
-                        defaultMessage='Save Edit Time'
-                    />
+                    {saving ? (
+                        <FormattedMessage
+                            id='save_button.saving'
+                            defaultMessage='Saving'
+                        />
+                    ) : (
+                        <FormattedMessage
+                            id='edit_post.time_limit_modal.save_button'
+                            defaultMessage='Save Edit Time'
+                        />
+                    )}
```

Comment:
This is new behavior, right? I guess intended, but wanted to double check.

Reply:
No, it's mostly the same as before. The only difference between this code and the previous version is that we're passing the correct default message for both IDs rather than always passing "Save Edit Time".

There's quite a few places where I had to move the ternary outside of the `id` field for a `FormattedMessage` for the tooling to detect it properly